Automattic/wp-calypso
# Checkout: Add CVV description

This adds a popover to the CVV field in checkout. Some people may not know what CVV means, so this helps them by explaining what and where to find it.

**Visa, MC, Discover**
<img width="730" alt="image" src="https://user-images.githubusercontent.com/448298/43111996-d8ac0bb2-8ec1-11e8-9a41-91f19e8c9642.png">

**AmEx**
<img width="732" alt="image" src="https://user-images.githubusercontent.com/448298/43208218-ea88891e-8ff7-11e8-999c-849caa59b799.png">

#### Testing

* Checkout this branch
* Make sure you don't have credits in your account
* Add an upgrade to your site
* On the checkout, assert the info popover is visible and works for the `CVV` field
* Enter a test credit card number, each time making sure the correct popover content is displayed
    * Visa - `4242424242424242`
    * MasterCard - `5555555555554444`
    * Discover - `6011111111111117`
    * AmEx - `378282246310005`
